src/statick_tool/plugins/tool/black.py

```diff
@@ -21,6 +21,12 @@ def get_file_types(self) -> list[str]:
         """Return a list of file types the plugin can scan."""
         return ["python_src"]
 
+    def get_binary(  # pylint: disable=unused-argument
```

Comment:
For all the `get_binary()` methods that do `return self.get_name()`, can you make this the method in `statick_tool.tool_plugin` module? Then you can remove it from all the tool plugins like this one.

Reply:
[66c84cc](https://github.com/sscpac/statick/pull/532/commits/66c84cc28d5261a96eafd4fabdfc0ddfdef2e4b2)